roda-common/roda-common-data/src/main/java/org/roda/core/data/v2/ip/disposal/DisposalConfirmationState.java

```diff
@@ -11,5 +11,5 @@
  * @author Miguel Guimarães <mguimaraes@keep.pt>
  */
 public enum DisposalConfirmationState {
-  PENDING, APPROVED, RESTORED, PERMANENTLY_DELETED, EXECUTION_FAILED
+  PENDING, APPROVED, RESTORED, PERMANENTLY_DELETED, EXECUTION_FAILED, SCHEDULED
```

Comment:
Why SCHEDULED state is on Disposal Confirmation?

Reply:
Forgot do remove